packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -31,3 +31,51 @@ export const userLogPayloadGuard = z.object({
 });
 
 export type UserLogPayload = z.infer<typeof userLogPayloadGuard>;
+
+export const signingKeyPayload = z.object({
+  keyId: z.string(),
+});
+
+export const signingKeyManagementPayloadGuard = z.object({
+  signing_algorithm: z.enum(['RS256']),
+  signing_keys: z.object({
+    previously_used_keys: signingKeyPayload.array(),
+    currently_used_key: signingKeyPayload,
+    next_key_in_queue: signingKeyPayload,
+  }),
+});
+
+export type SigningKeyManagementPayload = z.infer<typeof signingKeyManagementPayloadGuard>;
+
+export const apiAuthorizationSettingsPayloadGuard = z.object({
+  default_audience: z.string().url({ message: 'Invalid URL' }), // Default oidc audience (usually client_id).
+  default_directory: z.enum(['logto', 'email', 'sms', 'waad', 'adfs']), // Name of the connection to be use for Password Grant exchanges.
+});
+
+export type ApiAuthorizationSettingsPayload = z.infer<typeof apiAuthorizationSettingsPayloadGuard>;
+
+export const loginSessionManagementPayload = z.object({
```

Comment:
any reason for using `login`? since we are using `sign-in` globally

Reply:
if this is an `oidc-provider` config, shall we fix the values in v1?

Reply:
The issue for session management was added for later reference (LOG-1284).